### PR TITLE
Convert scripts/error-codes to use ES6 syntax

### PR DESCRIPTION
**Pull request by gkal19**
If need be, I can squash the commits.

### CODE REVIEW

File: scripts/error-codes/__tests__/replace-invariant-error-codes-test.js [hunk 1]
```diff
@@ -37,21 +37,13 @@ describe('error codes transform', () => {
     compare(
       "import invariant from 'shared/reactProdInvariant';\n" +
         "invariant(condition, 'Do not override existing functions.');",
-      "import _prodInvariant from 'shared/reactProdInvariant';\n" +
-        "import invariant from 'shared/reactProdInvariant';\n" +
-        '!condition ? ' +
-        '__DEV__ ? ' +
-        "invariant(false, 'Do not override existing functions.') : " +
-        `_prodInvariant('16') : void 0;`
+      `import _prodInvariant from 'shared/reactProdInvariant';\nimport invariant from 'shared/reactProdInvariant';\n!condition ? __DEV__ ? invariant(false, 'Do not override existing functions.') : ${`_prodInvariant('16') : void 0;`}`
```
Comment (gaearon) [inline scripts/error-codes/__tests__/replace-invariant-error-codes-test.js]: I don't think this is easier to read.

File: scripts/error-codes/codes.json [hunk 2]
```diff
@@ -1,252 +1,252 @@
 {
-  "0": "React.addons.createFragment(...): Encountered an invalid child; DOM elements are not valid children of React components.",
```
Comment (gaearon) [inline scripts/error-codes/codes.json]: This file is autogenerated. No need to change it.

Comment (gkal19) [reply scripts/error-codes/codes.json]: Sorry! I've forgotten to reverse this :cry:

**Comment by gkal19** [comment]
@gaearon what does that mean?
`Received return code 1 from: node ./scripts/prettier/index`

**Comment by gaearon** [comment]
You need to run `yarn prettier`, then commit.

**Comment by gkal19** [comment]
Sorry to keep you waiting.
I was away from my computer at the time.

**Comment by gaearon** [comment]
Looks good, thanks

**Pull request merged**